LBH.AdultSocialCare.Api/V1/UseCase/Common/Concrete/UpdateCarePackageUseCase.cs

```diff
@@ -27,8 +28,9 @@ public UpdateCarePackageUseCase(IMapper mapper, IDatabaseManager dbManager, ICar
 
         public async Task<CarePackagePlainResponse> UpdateAsync(Guid carePackageId, CarePackageUpdateDomain carePackageUpdateDomain)
         {
-            var package = await _carePackageGateway.CheckPackageExistsAsync(carePackageId, true);
-            var packageSettings = await _carePackageSettings.CheckPackageSettingsExistAsync(carePackageId, true);
+            var package = await _carePackageGateway.GetPackagePlainAsync(carePackageId, true).EnsureExists($"Care package with id {carePackageId} not found");
+            var packageSettings = await _carePackageSettings.GetPackageSettingsPlainAsync(carePackageId, true)
+                .EnsureExists($"Package settings for package with id {carePackageId} not found");
 
```

Comment:
```
    var package = await _carePackageGateway
        .GetPackagePlainAsync(carePackageId, true)
        .EnsureExists($"Care package with id {carePackageId} not found");

    var packageSettings = await _carePackageSettings
        .GetPackageSettingsPlainAsync(carePackageId, true)
        .EnsureExists($"Package settings for package with id {carePackageId} not found");
```

same comment about repeating patterns here :).

Approving for now, just please think about it some time, probably with this notion of repeating patterns variant above will become more readable to you one day :)